src/main/java/org/opensearch/timeseries/transport/ResultProcessor.java

```diff
@@ -499,7 +500,8 @@ private void executeAnalysis(
             Optional<Exception> previousException = nodeStateManager.fetchExceptionAndClear(configID);
             if (previousException.isPresent()) {
                 Exception exception = previousException.get();
-                LOG.error(new ParameterizedMessage("Previous exception of [{}]", configID), exception);
+                // TODO : {@link MultiEntityResultTests#testQueryErrorEndRunNotNow} fails if below line is uncommented
```

Comment:
What changes make the test fail?

Reply:
https://github.com/opensearch-project/OpenSearch/pull/19430

the original PR that above PR reverted was the cause, but since revert has been merged, i will uncomment this.